ddprof-ffi/src/exporter.rs

```diff
@@ -26,8 +26,8 @@ pub enum NewProfileExporterV3Result {
     Err(crate::Vec<u8>),
 }
 
-#[export_name = "ddprof_ffi_NewProfileExporterV3Result_dtor"]
-pub unsafe extern "C" fn new_profile_exporter_v3_result_dtor(result: NewProfileExporterV3Result) {
+#[export_name = "ddprof_ffi_NewProfileExporterV3Result_drop"]
+pub unsafe extern "C" fn new_profile_exporter_v3_result_drop(result: NewProfileExporterV3Result) {
```

Comment:
Minor: As we're going to rename this for consistency anyway (thanks!) may I suggest calling it `ddprof_ffi_ProfileExporterV3Result_drop`?

Indeed the type is called `NewProfileExporterV3Result` but that's a bit of a mouthful and we could rename it later as well :) (probably not in this PR, to avoid getting too off-topic). I actually think that since we're not too worried about backwards-compatibility right now we could even just rename it back to `ProfileExporter` without `New` or `V3` suffixes.

Reply:
I'll leave it for now -- we can shorten and clean this up later (like next PR, possibly).